static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -150,3 +151,83 @@ export function getDiseaseSymptomAssociation(
   }
   return rawData;
 }
+
+/**
+ * Fetches the chemical compound disease treatment data
+ * @param data
+ * @returns
+ */
+export function getCompoundDiseaseTreatment(
+  data: GraphNodes
+): CompoundDiseaseTreatmentData[] {
+  // checks if the data is empty
+  if (_.isEmpty(data)) {
+    return [];
+  }
+  const rawData: CompoundDiseaseTreatmentData[] = [];
+  // checks for null values
+  if (_.isEmpty(data.nodes) || _.isEmpty(data.nodes[0].neighbors)) {
```

Comment:
Yes! That's a good idea. So, if data is empty, so would be data.nodes. Are you suggesting to remove the `._isEmpty(data)` condition entirely because `_.isEmpty(data.nodes) ` does that for us?

Reply:
oh, no, I'm just saying you can do: `if (_.isEmpty(data) || _.isEmpty(data.nodes) || _.isEmpty(data.nodes[0].neighbors)) {return []}`

Reply:
Ah, got it! combined those together:)